eras/shelley/impl/cddl-files/shelley.cddl

```diff
@@ -1,258 +1,217 @@
-; Shelley Types
-
-block =
-  [ header
-  , transaction_bodies         : [* transaction_body]
-  , transaction_witness_sets   : [* transaction_witness_set]
-  , transaction_metadata_set   :
-      { * transaction_index => transaction_metadata }
-  ]; Valid blocks must also satisfy the following two constraints:
-   ; 1) the length of transaction_bodies and transaction_witness_sets
-   ;    must be the same
-   ; 2) every transaction_index must be strictly smaller than the
-   ;    length of transaction_bodies
-
-transaction =
-  [ transaction_body
-  , transaction_witness_set
-  , transaction_metadata / null
-  ]
+; This file was auto-generated from huddle. Please do not modify it directly!
+$hash28 = bytes .size 28
 
-transaction_index = uint .size 2
+$hash32 = bytes .size 32
```

Comment:
```
   For both type choices and group choices, additional alternatives can
   be added to a rule later in separate rules by using "/=" and "//=",
   respectively, instead of "=":

                 attire /= "swimwear"

                 delivery //= (
                 lat: float, long: float, drone-type: tstr
                 )

   It is not an error if a name is first used with a "/=" or "//="
   (there is no need to "create it" with "=").
```

So basically CDDL supports ad-hoc extending choices. Huddle does not support this. In any case, these definitions are the only definitions, so the use of `/=` is meaningless.